scripts/docs-generator/main.py

```diff
@@ -113,19 +120,16 @@ def add_to_module_to_page(mapping, prefix, module_to_page):
         if product == "unknown":
             continue
 
-        cloud_provider = resource.cloud_provider()
-        if cloud_provider is None or cloud_provider.value == "unknown":
+        if issubclass(resource, GCPResource) or issubclass(resource, GCPService):
+            out_path = f"gcp-{prefix}"
+        elif issubclass(resource, AWSResource) or issubclass(resource, AWSService):
+            out_path = f"aws-{prefix}"
+        elif issubclass(resource, KubernetesResource):
```

Comment:
should this include the kubernetes.ServiceContainer class?

Reply:
That's a subclass of KubernetesResource so should be good